webui/react/src/pages/ProjectNotes.tsx

```diff
@@ -56,23 +53,13 @@ const ProjectNotes: React.FC<Props> = ({ project, fetchProject }) => {
     [ProjectNoteDeleteModal, project?.id],
   );
 
-  const notesTabBarContent = useMemo(
-    () => (
-      <div className={css.tabOptions}>
-        <Button type="text" onClick={handleNewNotesPage}>
-          + New Page
-        </Button>
-      </div>
-    ),
-    [handleNewNotesPage],
-  );
-
-  useSetDynamicTabBar(notesTabBarContent);
+  useSetDynamicTabBar(<></>);
 
   return (
     <>
-      <PaginatedNotesCard
+      <Notes
         disabled={project?.archived}
+        multiple={true}
```

Comment:
should be able to simplify this to
```suggestion
        multiple
```